osbs/conf.py

```diff
@@ -554,3 +565,22 @@ def get_platform_node_selector(self, platform):
                 nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_platform_descriptors(self):
```

Comment:
This method is doing more than `get`. Let's name it something like: `resolve_platform_descriptors`
Alternatively, I think it's fine to not store the result in `self.platform_descriptors` and simply resolve and return the whole object, leaving it named as `get_platform_descriptors`.